NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -164,6 +164,8 @@
 if APPLYJETCORR and LEPTON_SETUP >=2022 :
     from ZZAnalysis.NanoAnalysis.modules.jetJERC import getJetCorrected
     insertBefore(reco_sequence, 'jetFiller', getJetCorrected(LEPTON_SETUP, DATA_TAG, IsMC, overwritePt=True))
+    from ZZAnalysis.NanoAnalysis.modules.jetVMAP import getJetVetoMap
+    insertBefore(reco_sequence, 'jetFiller', getJetVetoMap(LEPTON_SETUP, DATA_TAG))
```

Comment:
Question: the resulting sequence will be : [jetCorr,jetVeto,jetFiller] 
Is the jet veto intended to be run after the jetcorr module, ie over corrected jets, rather than on the original (uncorrected) jets?

Reply:
Thank you @namapane for pointing this out. I also discussed this with @AlessandroTarabini, and our assumption is that this is the correct sequence. Otherwise, since the corrections are already applied in NANOs, what I would have to do is remove the old corrections, apply the jet veto, and then apply the new corrections. I haven’t asked JME for confirmation on this but I can send them a message.